imap_processing/tests/hit/test_hit_l1a.py

```diff
@@ -29,6 +29,83 @@ def sci_packet_filepath():
     return imap_module_directory / "tests/hit/test_data/sci_sample1.ccsds"
 
 
+def test_validate_l1a_housekeeping_data(hk_packet_filepath):
+    """Validate the housekeeping dataset created by the L1A processing.
+
+    Compares the processed housekeeping data with expected values from
+    a validation csv file.
+
+    Parameters
+    ----------
+    hk_dataset : xr.Dataset
```

Comment:
hk_dataset is not passed into the function.

Reply:
forgot to update the doc string. Thanks for catching that!